lib/lightning/jobs/trigger.ex

```diff
@@ -54,7 +60,8 @@ defmodule Lightning.Jobs.Trigger do
         :custom_path,
         :type,
         :workflow_id,
-        :cron_expression
+        :cron_expression,
+        :has_auth_method
```

Comment:
Do we need `:has_auth_method` here? Since it's a virtual field, I don't see it being used anywhere other than in select queries..